### ISSUE TITLE
Getting Protractor to correctly interpret Pass/Fail results from the protractor-accessibility-plugin

### ISSUE THREAD

**Issue by JeffHall** [angular/protractor-accessibility-plugin#3]
As far as I can tell, the pass/fail status of Chrome-A11Y assertions are sent to STDOUT by the protractor-accessibility-plugin, separate from Protractor spec assertions themselves.  

In other words, if the Protractor specs that precede the Chrome-A11Y assertions succeed, the overall test result will always be positive, regardless of whether the Chrome-A11Y specs generated failure messages.

Does this make sense?  I was trying to use the protractor-console-plugin to manage the Chrome-A11Y results, but I don't think that is possible, since those results are essentially a separate entity from the Protractor specs themselves.

I'm a Protractor noob, so please forgive if this is a dumb question!

**Comment by sjelin** [comment]
I don't have this problem.  We even have a [test](https://github.com/angular/protractor-accessibility-plugin/blob/master/test.js#L20) which makes sure that failures actually cause an exit code of 1.  Am I misunderstanding your problem?

**Comment by JeffHall** [comment]
Aargh!  You're absolutely right @sjelin .  This issue was pure user error - looking for the right thing in the wrong place.  Closing issue...

Great plugin BTW - thanks to you and Marcy Sutton for all the great work!
